classes/MainMenu.cpp

```diff
@@ -18,97 +18,61 @@ MainMenu::MainMenu(Game* game) : Renderable(OVERLAY)
 	dim.x = LOGICAL_SCREEN_WIDTH / 2 - dim.w;
 	dim.y = 300;
 
-	mPlay_button = new Button("play", dim, this, OVERLAY, OVERLAY, PLAY);
+	auto play_button = new Button("play", dim, this, OVERLAY, OVERLAY, PLAY);
+	mButtons.push_back(play_button);
```

Comment:
Bitte in einer Zeile machen:
```suggestion
	mButtons.push_back(new Button("play", dim, this, OVERLAY, OVERLAY, PLAY));
```